PythonLessons/PyGame/pygame-lesson1.py

```diff
@@ -19,28 +19,29 @@
 BASIC_FONT = pygame.font.SysFont(None, 48)
 TEXT = BASIC_FONT.render('Hello world!', True, WHITE, BLACK)
 
-while True:
-    # Fill the screen to prevent any moving shapes leaving a 'trail'
-    WINDOW_SURFACE.fill(BLACK)
-
-    # Determine where to display TEXT
-    text_rect = TEXT.get_rect()
-    text_rect.centerx = WINDOW_SURFACE.get_rect().centerx
-    text_rect.centery = WINDOW_SURFACE.get_rect().centery
-    # Draw the text onto the surface
-    WINDOW_SURFACE.blit(TEXT, text_rect)
-
-    # Draw a circle, line and rectangle just to show how
-    pygame.draw.circle(WINDOW_SURFACE, (255, 0, 0), (10, 20), 30)
-    pygame.draw.line(WINDOW_SURFACE, (0, 0, 255), (200, 250), (225, 300))
-    pygame.draw.rect(WINDOW_SURFACE, (0, 255, 0), (150, 50, 100, 50))
-
-    # Update the screen
-    pygame.display.update()
 
+while True:
     # Check for user input
     for event in pygame.event.get():
         # Check to see if the "Close" button has been pressed
         if event.type == pygame.QUIT:
             pygame.quit()
             sys.exit(0)
+
+        # Fill the screen to prevent any moving shapes leaving a 'trail'
```

Comment:
This section below is now within the `for...loop`. Was that intended? It doesn't seem to use any `event`s.

Reply:
Using the example Greg said: 
`while true:
    for each event:
        check user input
    update game state and screen`
I agree, It doesn't seem like it should be inside the `for...loop`.

Reply:
Yes, the original intention was to move it below the `for...loop` not within it. Changed.